docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+   If you do not, try refreshing the page.
+
+   {{<imgproc src="/tutorials/helmet/synced-data.png" resize="x1000" declaredimensions=true alt="The data manager page in the Viam app, displaying three images of a person with a bounding box labeled NO-Hardhat around her face." >}}
+
+3. You can also try this with a hard hat on your head.
+
+   {{<imgproc src="/tutorials/helmet/hardhat-data.png" resize="x1000" declaredimensions=true alt="The data manager page in the Viam app, displaying three images of a person with a bounding box labeled NO-Hardhat around her face." >}}
+
+### Modify the detector config to sync only images without hard hats
+
+Until now, you've been identifying people without hard hats as well as people with hard hats.
+Now that you have verified that the detector and data sync are working, modify your config so that only images with people _without_ hard hats are captured:
+
+1. Navigate to your `objectfilter-cam` card on the **Config** tab.
+
+2. Delete the `"Hardhat",` line from the `"labels"` array.
+
+3. Click **Save config**.
+
+## Set up email notifications
+
+[Webhooks](https://en.wikipedia.org/wiki/Webhook) are a way to trigger a custom function to run when a certain event happens.
+For the purposes of this project, you will use a webhook to run a function that sends you an email when an image of someone without a hard hat is uploaded to the cloud.
+
+Before you configure a webhook on your machine, you need to create a serverless function for the webhook to call.
+
+### Create a serverless function
+
+A serverless function is a simple script that is hosted by a service such as [Google Cloud Functions](https://cloud.google.com/functions) or [AWS Lambda](https://aws.amazon.com/pm/lambda).
+You don't need to host it on your machine; instead, it is always available and runs only when an event triggers it.
+
+#### Set up your cloud function scaffold
+
+For this project, we used Google Cloud Functions, and wrote our function in Python.
+If you are new to cloud functions, you may find [this getting started guide](https://cloud.google.com/functions/docs/console-quickstart) useful.
+
+1. Create a Google Cloud (GCP) account.
+2. Create a project, then search for **Cloud Functions** in the Google Cloud console search bar.
+3. Click **CREATE FUNCTION**.
+4. Choose `1st gen` as the **Environment**.
+5. Give your function a name.
+   We called ours "hat-email."
+6. Choose your region.
+7. For trigger type, choose `HTTP`.
+8. Click **NEXT**.
+9. For **Runtime**, choose `Python 3.8`.
+10. Click **Deploy**.
+    The demo function won't do anything exciting yet; keep going through the next few sections before testing it.
+
+#### Configure email credentials
+
+To write a function that sends an email, you need to configure credentials to authenticate to the sending email account.
+For this project we used [SendGrid](https://sendgrid.com) (which has a free tier) to make configuration of your email notifications simpler:
```

Comment:
You want to introduce sendgrid first. You don't need credentials, you need something that can send emails. That service happens to have an integration for google cloud functions and that needs credentials. Bit putting the cart before the horse ;)
```suggestion
To write a function that sends an email, you need a service that can send emails.
For this project we used [SendGrid](https://sendgrid.com) (which has a free tier) to make configuration of your email notifications simpler.
Follow these instructiong to create a sendgrid account and configure the SendGrid API credentials for your Google Cloud Function:
```

Reply:
```suggestion
To write a function that sends an email, you need a service that can send emails.
For this project we used [SendGrid](https://sendgrid.com) (which has a free tier) to make configuration of your email notifications simpler.
Follow these instructions to create a SendGrid account and configure the SendGrid API credentials for your Google Cloud Function:
```